api/managementpb/backup/backups.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+package backup.v1beta1;
+
+option go_package = "api/managementpb/backup;backupv1beta1";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+
+// RetryMode specifies how backup should retry in case of failure.
+enum RetryMode {
+  RETRY_MODE_INVALID = 0;
+  AUTO = 1;
+  MANUAL = 2;
+}
+
+message PerformBackupRequest {
+  // Service identifier.
+  string service_id = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Machine-readable location ID.
+  string location_id = 2 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Database names to backup, If empty then each database is backed up.
+  // repeated string database_names = 3;
+  // If empty then name is auto-generated.
+  string name = 4;
+  // Human-readable description.
+  string description = 5;
+  // RetryMode retry_mode = 6;
+  // Retry interval in seconds. Default 30s.
+  // uint32 retry_interval = 7;
+  // How many times should backup be retried in case of failure. Default 2 times.
+  // uint32 retry_times = 8;
+
+}
+
+message PerformBackupResponse {
+  // Unique identifier.
+  string backup_id = 1;
+}
+
+// Backups service handles backup operations to DB.
+service Backups {
+  // PerformBackup request backup specified service to location.
+  rpc PerformBackup(PerformBackupRequest) returns (PerformBackupResponse) {
```

Comment:
Maybe it's better to name it `StartBackup`, `perform` a quite unusual verb for the method name.

Reply:
On arch design there is used verb `perform`.